lib/commands/app/create.ts

```diff
@@ -75,30 +86,51 @@ export default class AppCreateCmd extends Command {
 			AppCreateCmd,
 		);
 
-		const balena = getBalenaSdk();
-
-		// Create application
+		// Ascertain device type
 		const deviceType =
 			options.type ||
 			(await (await import('../../utils/patterns')).selectDeviceType());
-		let application: BalenaSDK.Application;
+
+		// Ascertain organization
+		const organization = options.organization || (await this.getOrganization());
+
+		// Create application
+		let application: Application;
 		try {
-			application = await balena.models.application.create({
+			application = await getBalenaSdk().models.application.create({
 				name: params.name,
 				deviceType,
-				organization: (await balena.auth.whoami())!,
+				organization,
 			});
 		} catch (err) {
 			// BalenaRequestError: Request error: Unique key constraint violated
 			if ((err.message || '').toLowerCase().includes('unique')) {
-				throw new ExpectedError(
-					`Error: application "${params.name}" already exists`,
-				);
+				const slug = `${organization.toLowerCase()}/${params.name.toLowerCase()}`;
+				throw new ExpectedError(`Error: application "${slug}" already exists`);
 			}
 			throw err;
 		}
-		console.info(
+
+		// Output result
+		console.log(
 			`Application created: ${application.slug} (${deviceType}, id ${application.id})`,
 		);
 	}
+
+	async getOrganization() {
+		const { getOwnOrganizations } = await import('../../utils/sdk');
+		const organizations = await getOwnOrganizations(getBalenaSdk());
+
+		if (organizations.length === 0) {
+			// User is not a member of any organizations (should not happen).
+			throw new Error('This account is not a member of any organizations');
+		} else if (organizations.length === 1) {
+			// User is a member of only one organization - use this.
+			return organizations[0].handle;
+		} else {
+			// User is a member of multiple organizations -
+			const { selectOrganization } = await import('../../utils/patterns');
+			return selectOrganization(organizations);
```

Comment:
This uses `getCliForm` from `'resin-cli-form'`, which I believe does not do auto TTY detection - and even if it does, there is still issue #1378. And these other issues:

* #1001
* #1043 
* #1089
* #1112
* #1378 
* #1808
* ...

And even attached JF support threads:
* https://jel.ly.fish/1418fb56-c0b7-4861-8ab1-75584e15f367
* https://jel.ly.fish/a675f6e4-6dcf-4de9-a566-a6e52da23104
* ...

I know there is the _"framework level"_ argument and the _"there is already this other prompt"_ argument and the _"it's beyond this PR"_ argument, but these arguments simply result in the problem being compounded by adding more prompts. I suggest a different approach: add a boolean flag like `--noninteractive` or `--notty` to `app create` that results in very simple behavior:

* If `--notty` is given, throw an error instead of raising an interactive prompt, e.g.:
```
throw new ExpectedError("This account has multiple organizations. Please specify an organization with the `-o` option")
```
* Otherwise, raise an interactive prompt.